.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 | Q             | A
 | ------------- | ---
-| Branch?       | 3.4 or master / 2.7, 2.8 or 3.3 <!-- see comment below -->
+| Branch?       | master / 2.7, 2.8, 3.3, or 3.4 <!-- see comment below -->
```

Comment:
this is going to need another change in 4 weeks to add 4.0, better wait?

Reply:
race condition :)

Reply:
I was thinking the same, but now that the PR is done...

Reply:
I think it's good to have merged it now. Another in 4 weeks, no big deal :)